### DISCUSSION TITLE
Sandboxie claims it won't sandbox until you pay.

### DISCUSSION THREAD

**Discussion by Rustbread** [sandboxie-plus/Sandboxie#3657] [General]
If you look for sandbox you expect to Applications to be separated from your System. And there is some sort of paid Privacy Mode that allows that. 

Thus why even bother to use Free Sanboxie if the first screen after running up application claims it will not protect your privacy until you pay. 
As it runs in native system language after translating it sounds like: "New Boxes with Enchanted Privacy that hide your files from Snadboxed Applications -> Pay Here". 

Are you telling me normally all "Sandboxed applications" have free reign to run through your files? What is the point then?

Cheers

**Comment by bastik-1001** [comment]
The privacy mode, which indeed requires a valid, active supporter certificate, has been added after the source code was released, and the same is true for the encryption feature, which requires a paid upgrade to use, something I understand to be problematic. I am not around to defend that, I just use the software and voluntarily contribute to the project.

Please take a look at the [project history](https://github.com/sandboxie-plus/Sandboxie?tab=readme-ov-file#-project-history). The original developer, Invincea and Sophos sold the software without these features. Now, features that were included in the sources, are free to use, some of the newly added features require some kind of contribution.

Maybe the wording could be improved, either in the English version, the translation or both, so it does not come across the way you perceived it.

You can use your favorite encryption software to create a container/volume to store your sandbox(es) in or on, that way they can't be accessed, when they are not mounted. By default, sandboxes are stored at C:\Sandboxes\, but the path can be changed per sandbox, which enables you to use an encrypted container, a removable drive (possibly even encrypted) or a folder in some user to restrict access to files within a sandbox or sandboxes.

The original motivation, as far as I can tell, was to prevent changes to the real system, unless the user made it possible by choice, privacy was not the goal. By default, the sandboxed process has read access to whatever the user can read. Partly as it is required to run installed software in a sandbox without having the user to configure it. Originally, Sandboxie allowed sandboxed processes to read the memory of unsandboxed processes, which is not the case anymore since version 1.0.16 / 5.55.16 (unless configured to allow it).

With Sandboxie (and Sandboxie-Plus) you can use resource access settings to control (and thereby deny, if you wish) access to resources (just like the name suggests), without having to pay. This is less convenient than the privacy mode, of course.
